protocol/communities_events_token_master_test.go

```diff
@@ -166,7 +70,7 @@ func (s *TokenMasterCommunityEventsSuite) TestTokenMasterCannotDeleteBecomeToken
 func (s *TokenMasterCommunityEventsSuite) TestTokenMasterAcceptMemberRequestToJoin() {
 	community := setUpOnRequestCommunityAndRoles(s, protobuf.CommunityMember_ROLE_TOKEN_MASTER, []*Messenger{})
 	// set up additional user that will send request to join
-	user := s.newMessenger("", []string{})
+	user := s.newMessenger("somePassword", []string{"0x0123400000000000000000000000000000000000"})
```

Comment:
@osmaczko I think it's not a good thing to use any constants in tests. Because in theory the code might be working fine with this particular constant, but failing with another value. 

It sounds like an overkill for `password` or `displayName`, but I think it's a good approach anyway.

So I see this as "legal, even required, flakiness". 
And randomness shouldn't be too much time execution on top, if you worry about that (I do worry about that 😄).

Reply:
That's fair, but I always believe that tests must be deterministic. We could address the points you mentioned with fuzzy testing if needed, which will also be deterministic if the same seed is persisted (when the same seed is provided, the fuzzing engine will generate the same sequence of inputs each time).